### ISSUE TITLE
GravityProperty validValues needs refinement

### ISSUE THREAD

**Issue by arouinfar** [phetsims/density-buoyancy-common#311]
For https://github.com/phetsims/buoyancy/issues/51

Over in https://github.com/phetsims/density-buoyancy-common/issues/270 MaterialProperty got validValues, and in #308 we uninstrumented `materialProperty.customMaterial` when custom was not a valid value.

I'd like to follow a similar pattern for `gravityProperty.customGravity` but discovered that custom is always a valid value for gravity, even in sims/screens where the gravity is limited to just Earth.

For all sims/screens where there isn't a Gravity control:  

Uninstrument model.gravityProperty:
- [ ] All of Buoyancy: Basics
- [ ] All of Density

Valid value only Earth:
- [ ] Buoyancy -- All but Lab

Once that's done...
- [ ] Uninstrument `customGravity` if custom is not a valid value, just like #308.

Assigning to @samreid since he addressed #308.

**Comment by samreid** [comment]
> For all sims/screens where there isn't a Gravity control, the only validValue should be Earth.

I just want to double check the design before starting on this issue. A Property with only one validValue is equivalent to a read-only Property. @zepumph and I have discussed that sometimes it is preferable to uninstrument a read-only Property, unless there is an important use case for the read-only access.

@arouinfar what do you recommend?

**Comment by arouinfar** [comment]
Great question @samreid. 

The gravity of Earth is not a constant in this suite, as the public `gEarth` query parameter can be used by phet-brand users to set g to a rounder number like 10 m/s<sup>2</sup>. People often make this approximation so weights are nicer numbers, which is especially helpful for younger audiences. We added this query parameter based on user feedback. For PhET-iO, there needs to be user-settable Property to allow for this customization, which we have with `global.model.gravities.earth.gravityValueProperty`. Would clients still be able to change Earth's gravity if `gravityProperty` is made read-only?

The implementation of gravities and materials is very similar, so I suggested limiting the validValues, because that is what we did for the fluids. In Density, for example, the only validValue for `fluid.materialProperty` is water.
